### ISSUE TITLE
Docstring needs updating

### ISSUE THREAD

**Issue by GabrielJMS** [Dessia-tech/plot_data#336]
https://github.com/Dessia-tech/plot_data/blob/131f369a7374485858593b3d3af13fbf1c8a324a/plot_data/core.py#L1101

**Comment by Tanguylo** [comment]
Is this ok for you ? 
```
:param clockwise: True if you want the arc the be drawn clockwise, False otherwise
:type clockwise: bool
```

**Comment by Tanguylo** [comment]
Resolved in 0.22
